src/main/java/com/blackducksoftware/tools/commonframework/core/config/ConfigurationProperties.java

```diff
@@ -73,6 +73,12 @@ public Properties getProperties() {
 		final Iterator<String> iter = config.getKeys();
 		while (iter.hasNext()) {
 			final String key = iter.next();
+			if (key.equals("cc.password")){
+				final String unEscapedValue = unescape(config.getString(key));
+				logger.debug("getProperties(): including: " + key + "=" + "******");
+				propertiesObject.put(key, unEscapedValue);
+				continue;
+			}
 			final String unEscapedValue = unescape(config.getString(key));
```

Comment:
1. This approach duplicates the unescape() and the propertiesObject.put. I'd suggest putting only the logger.debug() inside a conditional.
2. We can make this even better. This class actually processes multiple types of passwords (Code Center, Protex, database, etc.). The keys always end in ".password", so I'd suggest replacing key.equals("cc.password") with something like key.endsWith(".password"), with will benefit all types of passwords.